Gandalf/build.sh

```diff
@@ -1,36 +1,81 @@
 #!/bin/bash
-# 
+#
 # Packages "Gandalf" into a deb
 #
 # Follow https://google.github.io/styleguide/shell.xml
 # for the most part (You can ignore some, like error checking for mv)
-echo "We're trying something new! Put in the version you want in version.txt (either as 'x' or 102) and we'll have one build.sh script!"
-echo "sleeping for 5 seconds"
-sleep 5
-VER=$(cat version.txt)
+# Specify version
 
-# Config
 PKG_VERSION="2.5.5" #Bump this everytime you update something.
-CONFLICTS_FILE="$VER/conflicts.txt"
-NAME=$(cat $VER/name.txt)
-FIRM=$(cat $VER/firmware.txt)
+
+# Define script styling (tput)
+
+BOLD=$(tput bold)
+NORMAL=$(tput sgr0)
+RED=$(tput setaf 1)
+
+# Check if started without parameters
+if [ "${1}" = "" ]; then
```

Comment:
https://google.github.io/styleguide/shell.xml

`${1}` should be `$1`.

Also, I believe you're declaring a variable.